WiserTaskScheduler/WiserTaskScheduler/Core/Services/ErrorNotificationService.cs

```diff
@@ -40,6 +46,19 @@ public async Task NotifyOfErrorByEmailAsync(string emails, string subject, strin
         {
             return;
         }
+        
+        // Generate SHA 256 based on configuration name, time id, order id and message
+        var sha256 = System.Security.Cryptography.SHA256.Create();
```

Comment:
Please add this to a `using` block. The `SHA256` class implements `IDisposable` and the general rule of thumb is that you should always make sure you dispose those classes, to make sure that they can't cause memory leaks.